tests/python/gpu/test_operator_gpu.py

```diff
@@ -1359,17 +1359,37 @@ def test_rnn_layer():
 def test_sequence_reverse():
     check_sequence_reverse(mx.gpu(0))
 
-@unittest.skip("Test fails intermittently. Temporarily disabled until fixed. Tracked at https://github.com/apache/incubator-mxnet/issues/8211")
 def test_autograd_save_memory():
-    x = mx.nd.zeros((128, 512, 512), ctx=mx.gpu(0))
+    """test_autograd_save_memory.
+    Flaky, may cause OOM. Tracked at https://github.com/apache/incubator-mxnet/issues/8211"""
+    import platform
+
+    def get_mem():
```

Comment:
Is this a good way to track memory?
Its also platform dependent, what is the purpose of this test?

https://stackoverflow.com/questions/131303/how-to-measure-actual-memory-usage-of-an-application-or-process#comment56267191_131303

Reply:
This is to test whether autograd memory-save fix is working or not. Given that our CI slaves haven't been homogenous (of which the reason eludes me), the hard-coded number may result in a max memory usage above available mem. So after this change, instead of testing whether max number causes OOM, it tests whether the increment is within limit. I think this test being platform-dependent is OK since the path being tested is not necessarily platform dependent.